test/e2e/run_networking_test.go

```diff
@@ -376,6 +376,12 @@ var _ = Describe("Podman run networking", func() {
 		Expect(session.ExitCode()).To(Equal(0))
 	})
 
+	It("podman run slirp4netns network with mtu", func() {
+		session := podmanTest.Podman([]string{"run", "--network", "slirp4netns:mtu=9000", ALPINE, "ping", "-c1", "10.0.2.2"})
+		session.Wait(30)
+		Expect(session.ExitCode()).To(Equal(0))
+	})
```

Comment:
The test should check if the mtu was set correctly. For example run `ip addr` instead of `ping` in the container and check with
```golang
Expect(session.OutputToString()).To(ContainSubstring("mtu 9000"))
```

Reply:
Done. However, I modelled the test on current ones, i.e.: cdir and loopback. Maybe those need a touch up.